Content.Server/Traits/Assorted/ParacusiaComponent.cs

```diff
@@ -0,0 +1,23 @@
+namespace Content.Server.Traits.Assorted;
+
+[RegisterComponent]
+public sealed class ParacusiaComponent : Component
+{
+    /// <summary>
+    /// The random time between incidents, (min, max).
+    /// </summary>
+    [DataField("timeBetweenIncidents", required: true)]
+    public Vector2 TimeBetweenIncidents { get; }
```

Comment:
Don't try to be clever like this, just do it as 2 floats if you really don't want it as 2 doubles.

Reply:
> Don't try to be clever like this, just do it as 2 floats if you really don't want it as 2 doubles.

last i checked using tuples just.. didn't mix with the serializer. I'm using the same pattern in OR code due to that.

Reply:
I'd just make it 2 separate datafields.

If it were me coding it I'd just make it 2 timespans as the savings is negligible, especiallly for making the API more painful, but it's eh.

Reply:
> I'd just make it 2 separate datafields.
> 
> If it were me coding it I'd just make it 2 timespans as the savings is negligible, especiallly for making the API more painful, but it's eh.

in this case it should def be two timespans yes. I mean in the scenario of "i want a range of values"